eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -14,20 +14,15 @@ parameters:
 steps:
 - ${{ if and(ne(parameters.archType, 'arm'), ne(parameters.archType, 'arm64')) }}:
 
-<<<<<<< HEAD
   # For NativeAOT-LLVM, we have just built the Wasm-targeting native artifacts (the runtime and libraries).
   # Now we need to build the cross-targeting compilers, RyuJit and ILC.
 
   - ${{ if eq(parameters.archType, 'wasm') }}:
-    - script: $(Build.SourcesDirectory)/build$(scriptExt) clr.alljits+clr.wasmjit+clr.tools -c $(buildConfigUpper)
+    - script: $(Build.SourcesDirectory)/build$(scriptExt) clr.alljits+clr.wasmjit+clr.nativeaotlibs+clr.tools -c $(buildConfigUpper)
```

Comment:
Should this be mentioned in the documentation too?

It is a bit of shame one needs to build the NAOT libs now for a test project to compile, though I suppose it doesn't take that much time.

Reply:
Indeed it is a shame.  Doc updated.